docs/latest/tutorial/tutorial-main-renderer.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Main and Renderer Process Communication'
+description: 'This guide will step you through the process of creating a barebones Hello World app in Electron, similar to electron/electron-quick-start.'
+slug: node-access
+hide_title: false
+---
+
+:::info Tutorial parts
+This is part 3 of the Electron tutorial. The other parts are:
+
+1. [Prerequisites]
+1. [Scaffolding]
+1. [Main and Renderer process communication][main-renderer]
+1. [Adding Features][features]
+1. [Application Distribution]
+1. [Code Signing]
+1. [Updating Your Application][updates]
+
+:::
+
+At the end of this part you will have an Electron application that uses
+a `preload` script to safely expose features from the main process
+into the renderer one.
+
+## Main and Renderer process communication
+
+Electron's main process is a Node.js process that has full OS access.
+On top of [Electron's modules][modules], you can also access [Node.js ones][node-api],
+as well as any compatible code downloaded via `npm`.
+
+If you are not familiar with Node.js, we recommend you to first
+read [this guide][node-guide] before continuing.
+
+## Using `contextBridge` and a preload script
+
+In Electron, the main process is the one that has access to Node.js, and the
+renderer process to the web APIs. This means it is not possible to access the
+Node.js APIs directly from the renderer process, nor the DOM from the main process.
+They're in entirely different processes with different APIs!
+
+:::tip Further reading 📚
+This is your last reminder to look at [Electron's Process Model][process-model] to
+better understand all of this!
+:::
+
+This is where attaching a [`preload` script][preload-script] to your
+renderer comes in handy. A `preload` script runs before the renderer process is loaded,
+and has access to both renderer globals (e.g. `window` and `document`) and a Node.js
+environment.
```

Comment:
If we're sticking with `sandbox: true`,  we should link the preload section of the Sandboxing doc.

https://www.electronjs.org/docs/latest/tutorial/sandbox#preload-scripts

Reply:
Done